scripts/support/create_ge_images/create_secgroup

```diff
@@ -23,13 +23,15 @@
 #
 
 
-nova secgroup-create allopen
+export PATH=/usr/bin:$PATH ; export PYTHONPATH=/usr/lib/python2.7/dist-packages
```

Comment:
This is not an issue in this case; the scripts are only supported for a specific Linux distribution (Ubuntu 14.04).

Why a distribution is chosen instead of using a virtualenv? Because this is a shell scripts that requires more software than python packages. Requirements are not defined by a pip requirements.txt, but for apt-get packages.

The PATH is defined to avoid using packages installed with pip. Of course, the scripts could be written to use also virtualenv, but here virtualenv does not provide any advantage to the user, and it is more easy to install the packages with apt-get.

An important point is also that the test scripts of the images are invoked in local and therefore a specific distribution must be chosen to provide a standard environment.

The only exception is the publish script (an independent tool that can be installed in another location), because it is python code that has its own dependencies (the osclient ones) and therefore it uses a virtualenv.